modules/EmbedPlayer/resources/mw.EmbedPlayerNative.js

```diff
@@ -180,8 +180,11 @@
             this.bindHelper('switchAudioTrack' + this.bindPostfix, function (e, data) {
                 _this.switchAudioTrack(data.index);
             });
-            this.bindHelper('liveOnline' + this.bindPostfix, function () {
-                _this.load();
+            this.bindHelper('liveOffline' + this.bindPostfix, function () {
+	            this.bindOnceHelper('liveOnline' + this.bindPostfix, function () {
```

Comment:
Is there a chance that we get liveOffline couple of times and then each time do a bind once?

Reply:
for the same offline scenario? no.

Reply:
Probably yes, wondering if we need a state flag as well but it gets ugly cause we also have it in hlsjs as well.